apps/prairielearn/python/traverse.py

```diff
@@ -52,6 +52,9 @@ def format_attrib_value(v: str) -> str:
 
 
 def get_source_definition(element: lxml.html.HtmlElement) -> str:
+    if type(element.tag) is not str:
```

Comment:
Is the use of `isinstance` not appropriate here? I'm actually surprised ruff doesn't complain in this case.

Reply:
I think we haven't merged the PR that enables more of the ruff checks?

This is just me doing stupid things, `isinstance(...)` is of course better. Changed in 852ee9d.